src/offset_of.rs

```diff
@@ -145,4 +173,25 @@ mod tests {
 
         assert_eq!(offset_of!(sub::Foo, x), 0);
     }
+
+    #[test]
+    fn generic_offset() {
+        #[repr(C, packed)]
```

Comment:
Please test unpacked structs like most other tests do; we already have a test specifically for packed structs.

Reply:
Sorry, it's a bad habit